lib/contentful_middleman/helpers.rb

```diff
@@ -28,10 +29,11 @@ def localize_array(value, locale, fallback_locale='en-US')
     end
 
     def localize_value(value, locale, fallback_locale='en-US')
-      if value.respond_to? :fetch
-        return value.fetch(locale) if value.key? locale
-        return value.fetch(fallback_locale) if value.key? fallback_locale
-      end
+      value = value.fetch(locale) if value.respond_to?(:fetch) && value.respond_to?(:key) && value.key?(locale)
```

Comment:
`key` checks are unnecessary and should be `key?` if they were.

The call to `localize_array` and `localize_entry` are totally spot-on. Hadn't tested nested localized items, and it's a great addition.